contents/centos-stream.mdx

```diff
@@ -0,0 +1,374 @@
+---
+title: CentOS Stream 软件仓库镜像使用帮助
+cname: 'centos-stream'
+---
+
+该文件夹只提供 CentOS Stream 9 ，如果需要非 Stream 版的 CentOS ，请参考 centos 的帮助。
+
+CentOS Stream 9 默认启用了包管理工具 dnf ，其是 yum 包管理工具的替代品。dnf 与 yum 大部分的命令都是通用的，dnf 也使用/etc/yum.repos.d/进行镜像配置。
+
+CentOS Stream 9 中源被整合入两个文件 centos.repo 和 centos-addons.repo ，由于文件中不包含 baseurl 字段，需要手动插入，通过文本替换修改源的方法较为复杂，也可以选择直接复制最后的替换结果覆盖源文件。
+
+### 文本替换
+
+将这段代码保存为一个文件，例如 update_mirror.pl 。
+
+<CodeBlock>
```

Comment:
Sorry for using English, my IME is broken. 

Maybe you can add `enableQuickSetup` and `filepath="/tmp/update_mirror.pl"`